python/lsst/ts/Dome/dome_csc.py

```diff
@@ -465,61 +464,64 @@ async def statusAMCS(self):
 
         This command will be used to request the full status of the AMCS lower level component.
         """
-        await self.request_and_send_llc_status(LlcName.AMCS, self.tel_domeADB_status)
+        await self.request_and_send_llc_status(LlcName.AMCS, self.tel_azimuth)
 
     async def statusApSCS(self):
         """ApSCS status command not to be executed by SAL.
 
         This command will be used to request the full status of the ApSCS lower level component.
         """
-        await self.request_and_send_llc_status(LlcName.APSCS, self.tel_domeAPS_status)
+        await self.request_and_send_llc_status(LlcName.APSCS, self.tel_apertureShutter)
 
     async def statusLCS(self):
         """LCS status command not to be executed by SAL.
 
         This command will be used to request the full status of the LCS lower level component.
         """
-        await self.request_and_send_llc_status(LlcName.LCS, self.tel_domeLouvers_status)
+        await self.request_and_send_llc_status(LlcName.LCS, self.tel_louvers)
 
     async def statusLWSCS(self):
         """LWSCS status command not to be executed by SAL.
 
         This command will be used to request the full status of the LWSCS lower level component.
         """
-        await self.request_and_send_llc_status(LlcName.LWSCS, self.tel_domeLWS_status)
+        await self.request_and_send_llc_status(LlcName.LWSCS, self.tel_lightWindScreen)
 
     async def statusMonCS(self):
         """MonCS status command not to be executed by SAL.
 
         This command will be used to request the full status of the MonCS lower level component.
         """
-        await self.request_and_send_llc_status(LlcName.MONCS, self.tel_domeMONCS_status)
+        await self.request_and_send_llc_status(LlcName.MONCS, self.tel_interlocks)
 
     async def statusThCS(self):
         """ThCS status command not to be executed by SAL.
 
         This command will be used to request the full status of the ThCS lower level component.
         """
-        await self.request_and_send_llc_status(LlcName.THCS, self.tel_domeTHCS_status)
+        await self.request_and_send_llc_status(LlcName.THCS, self.tel_thermal)
 
     async def request_and_send_llc_status(self, llc_name, telemetry_function):
         command = f"status{llc_name.value}"
         status = await self.write_then_read_reply(command=command, parameters={})
         # Store the status for unit tests.
-        self.lower_level_status[llc_name.value] = status[llc_name.value]
-        self.log.info(status)
-        self.convert_telemetry_radians_to_degrees_and_send(
-            status[llc_name.value], telemetry_function, llc_name
-        )
+        self.lower_level_status[llc_name.value] = status[llc_name.value][0]
```

Comment:
Since there is no conversion to radians involved with the light and wind screen, I have removed the convert_telemetry_list_radians_to_degrees_and_send method and simplified the code quite a bit.